dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBean.java

```diff
@@ -156,6 +187,16 @@ public static ExternalTool parseAddExternalToolManifest(String manifest) {
         return new ExternalTool(displayName, description, type, toolUrl, toolParameters, contentType);
     }
 
+    // -------------------- PRIVATE --------------------
+
+    private boolean isFilePublic(DataFile file, DatasetVersion datasetVersion) {
```

Comment:
I believe this is intentional but just want to be sure: we only check if given file is public in specific dataset version and not across all versions? 
The consensus on a meeting was that we are ok with this being inconsistent with files endpoint?

Reply:
AFAIK we decided to check only in context of the version the user is viewing.